### ISSUE TITLE
Empty Results

### ISSUE THREAD

**Issue by dingdong42564** [Krazete/antscoper#2]
It no longer grabs schedules from the registrar. Not sure why. All classrooms at all times are showing as empty.

**Comment by Krazete** [comment]
Figured out what went wrong. I remember WebSOC text results being like this:
```
Art      1C        ART IN CONTEXT
    CCode Typ Sec Unt   Instructor         Time                     Place      Final                      Max      Enr      WL Req Rstr Status 
    01020 LEC A   4     LAFARGE, A.        TuTh         5:00- 6:20p HG 1800    Thu, Jun 12, 4:00-6:00pm    131     131     n/a 333      FULL   
```
And checking it now, it's like this:
```
Art      11A       HISTORY CONTEMP ART
    CCode Typ Sec Unt  Instructor         Modality               Time      Place Max    Enr Req  Rstr    Status
    01038 LEC A   4    CARSON, J.         In-Person              Th 1:00- 3:20p ART 160    33   33 62           FULL
```
The `websoc/parse.py` script identifies time and place by looking at where `Time`, `Place`, and `Max` is in the table header, expecting everything to be lined up neatly so it could use those indices for the following rows. But they added a new `Modality` column and neglected to adjust the formatting. With the alignment all screwed up now, my script thinks `Art 11A` is located at `:20p A` instead of at `ART 160`.
I'll have to figure out some other way to parse out time and place.

**Comment by Krazete** [comment]
Alright, I fixed the parser problem in 570a5d4.
There was also an issue with my scraper returning "Remote end closed connection without response" (WebSOC probably tightened request limits a bit), so I added a delay in 78d82c2 to fix that.
It should be working properly now. Thank you for the report.
